package.json

```diff
@@ -2147,7 +2135,7 @@
         "mocha-junit-reporter": "^2.0.0",
         "mocha-multi-reporters": "^1.5.1",
         "prettier": "^2.3.2",
-        "prettier-plugin-sh": "^0.7.1",
+        "prettier-plugin-sh": "^0.6.1",
```

Comment:
Thinking you're slightly off the tip of `master`? Is there a reason to regress?

Reply:
see prior comment